main.go

```diff
@@ -19,21 +21,34 @@ func failf(format string, args ...interface{}) {
 	os.Exit(1)
 }
 
+func stopInstance(wg *sync.WaitGroup, uuid string) {
+	defer wg.Done()
+
+	cmd := command.New("gmsaas", "instances", "stop", uuid)
+	out, err := cmd.RunAndReturnTrimmedCombinedOutput()
+	if err != nil {
+		failf("Failed to stop an instance, error: error: %s | output: %s", cmd.PrintableCommandArgs(), err, out)
```

Comment:
Maybe it worth mentioning the instance uuid here? 

`Failed to stop instance <UUID>, error: error: %s | output: %s` or something.